datalad/support/tests/test_sshconnector.py

```diff
@@ -167,3 +170,22 @@ def test_ssh_copy(sourcedir, sourcefile1, sourcefile2):
             eq_(content, fp.read())
 
     ssh.close()
+
+
+@skip_ssh
+def test_ssh_compound_cmds():
+    ssh = SSHManager().get_connection('ssh://localhost')
+    out, err = ssh('[ 1 = 2 ] && echo no || echo success')
```

Comment:
You can feed it any valid Python string. As the docs say, quoting (via. sh_quote) is up to the user. In any case, this string is passed as the one (and single) last argument to the SSH call -- no matter how many unmatched quotes it contains. If it does, then the call on the remote will fail, of course -- but it will be on the remote.